.github/actions/setup-rust/action.yml

```diff
@@ -17,6 +17,12 @@ inputs:
     default: ""
 
 outputs:
+  bench-packages:
+    description: Benchmarkable packages for the current OS
+    value: ${{
+      (runner.os == 'Linux' && '--help') ||
```

Comment:
Couldn't find any other way to say "no packages"

Reply:
Haha that is quite creative!